docs/fedora-releases-guide.md

```diff
@@ -18,6 +18,83 @@ Doing Fedora releases with Packit means utilising these jobs:
 
 Every job takes care of a different part of the release process.
 
+## Propose downstream and pull from upstream comparison
+### Debugging via CLI
```

Comment:
I am not sure if this fits here -- the following table is not only about debugging via CLI. I would freely delete this.

```suggestion
```